src/pages/components/menu.mdx

```diff
@@ -199,18 +227,28 @@ import MenuSizeCode from '!!raw-loader!../../examples/dropdowns/menu/Size.tsx';
 
 <Component components={props.data.mdx.components} componentName="menu" />
 
+A Menu manages placement, positioning and layout of the child [Item](#item) components.
+
 <PropSheet components={props.data.mdx.components} componentName="menu" />
 
 ### NextItem
 
 <Component components={props.data.mdx.components} componentName="nextItem" />
 
+The NextItem component displays the next icon to indicate the presence of a succeeding level. It enables
+additional RTL-aware keyboard navigation in nested menus. The [Dropdown](#dropdown) component's `isOpen` and `onStateChange`
+props can be used to conditionally handle the open/close state based on selection.
+
```

Comment:
Fleshing this part out is great! Many consumers mistakenly nest items in `Select` components. Providing some content around `NextItem` helps emphasize that nesting structure belongs in `Menu` rather than `Select`.